conf/configSoupX.R

```diff
@@ -1,9 +1,9 @@
 ### --------------Initial info----------------------------
 PROJECT = "Mouse Blood project" ## set project name
 ORGAN = 'Blood'           #For external annotation. Options: Blood, Heart, Intestine, Kidney
-SPECIES = "Mouse"         #For external annotation. Options: Human, Mouse
-MCA_NAME = "Bone-Marrow" #For MCA annotation.      Options: check http://bis.zju.edu.cn/MCA/
```

Comment:
For this file. I would keep the old config.R, rather than only rename it to the configSoupX.R